src/components/Feed/ProposalCreator.js

```diff
@@ -54,23 +55,33 @@ function ProposalCreator({ proposal }) {
         display: flex;
       `}
     >
-      <div>
-        {profile?.image ? (
-          <img
-            src={profile.image}
-            height="43"
-            width="43"
-            alt=""
-            css={`
-              border-radius: 50%;
-              display: block;
-              object-fit: cover;
-            `}
-          />
-        ) : (
-          <EthIdenticon address={proposal.creator} radius={50} scale={1.8} />
-        )}
-      </div>
+      <Link to={`/profile?account=${proposal.creator}`}>
```

Comment:
With the previous comment, this should be now 
```suggestion
      <Link href={`/profile?account=${proposal.creator}`} external={false}>
```